tests/ocean_merge/data/input.nml

```diff
@@ -0,0 +1,8 @@
+     &mask_nml
+     ocean_mask_dir="/ocean/mask/dir"
+     ocnres="mx025"
```

Comment:
Should this be mx500 as per reg_tests/grid_gen/c96.uniform.sh LIne 20 "ocn = 500"?

Reply:
Done at 6364736.